packages/components/src/tools-panel/stories/index.js

```diff
@@ -138,19 +142,36 @@ export const WithNonToolsPanelItems = () => {
 export const WithOptionalItemsPlusIcon = () => {
 	const [ height, setHeight ] = useState();
 	const [ width, setWidth ] = useState();
+	const [ minWidth, setMinWidth ] = useState();
 
 	const resetAll = () => {
 		setHeight( undefined );
 		setWidth( undefined );
+		setMinWidth( undefined );
 	};
 
+	const includeDefaultControls = boolean( 'includeDefaultControls', false );
+
 	return (
 		<PanelWrapperView>
 			<Panel>
 				<ToolsPanel
 					label="Tools Panel (optional items only)"
 					resetAll={ resetAll }
+					key={ includeDefaultControls }
```

Comment:
Is `includeDefaultControls` used as a `key` to force `ToolsPanel` to re-render?

Reply:
Yes. Unfortunately without it, the panel doesn't refresh properly. At first I thought it pointed to a bug in the source code, but the storybook and the Gutenberg app are behaving differently here. 🤷